tests/puppeteer/campaigns/utils/globals.js

```diff
@@ -18,7 +18,7 @@ global.INSTALL = {
 global.BROWSER_CONFIG = {
   headless: JSON.parse(process.env.HEADLESS || true),
   timeout: 0,
-  slowMo: 25,
+  slowMo: 5,
```

Comment:
👍

Reply:
Oh damn. That's a problem...

Reply:
I will try something else but in a different PR